### DISCUSSION TITLE
A way to download all captured video links at once?

### DISCUSSION THREAD

**Discussion by NevGi-cpu** [subhra74/xdm#617] [General]
Is there a way to download all captured links at one go. You know that the browser monitor keeps capturing the links of the video that we have played which we can see when we click the browser extension. [View](https://prnt.sc/1ck1b71)
So is there a way to download those stored videos links at one go rather than having to go clickling and downloading each video at a time?

**Comment by GODK1NG** [comment]
Would really like this option to be included in the October update as well

Would be good to also include the option in the floating "DOWNLOAD VIDEO" button, maybe next to "More formats" after the button/menu has been expanded
